### PR TITLE
chore: change token in readme to constant

### PR DESCRIPTION
**Pull request by jeongnaehyeok**
**Please describe the changes this PR makes and why it should be merged:**
I was confused when I saw the guide code in README.md. Cuz **'CLIENT_ID'** is a constant expression but **'token'** is not, I pondered for a moment what was wrong. I think this should be expressed in the same way. So I modify this.
**Status and versioning classification:**
Modify **'token'** to **'TOKEN'** in all example codes

### CODE REVIEW

**Comment by jaw0r3k** [comment]
um as u said to 'TOKEN'
not to variable

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ const commands = [
 	},
 ];
 
-const rest = new REST({ version: '10' }).setToken('token');
+const rest = new REST({ version: '10' }).setToken(TOKEN);
```
Comment (jaw0r3k) [inline README.md:69]:
```suggestion
const rest = new REST({ version: '10' }).setToken('TOKEN');
```

Comment (jaw0r3k) [reply README.md:69]: hmm idk

Comment (jeongnaehyeok) [reply README.md:69]: I think it would be good to use it as one expression. This confuses beginners.

**Review by iCrawl — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**